.pre-commit-config.yaml

```diff
@@ -0,0 +1,19 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.4.0
+    hooks:
+      - id: trailing-whitespace
+  - repo: https://github.com/digitalpulp/pre-commit-php.git
+    sha: 1.3.0
+    hooks:
+      - id: php-lint
+  - repo: local
+    hooks:
+      - id: phpcbf
+        name: phpcbf
+        entry: vendor/bin/phpcbf src
+        language: script
+      - id: phpcs
+        name: phpcs
+        entry: vendor/bin/phpcs src
+        language: script
```

Comment:
Would these commands have problem with powershell?

Reply:
hmmm it does have a problem with powershell, but i'm able to run it on Windows' cmd